src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -10,7 +10,7 @@
 public class Tag {
 
     public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
-    public static final String VALIDATION_REGEX = "\\p{Alnum}+";
+    public static final String VALIDATION_REGEX = ".+";
```

Comment:
Why the change to accept all characters though

Reply:
I guess just so that I can have dashes in my group names lol

Reply:
Because one of your invalid tags in your sample data was `friend*` but that's true I'll added dashes and underscores in but no special characters